.circleci/config.yml

```diff
@@ -239,459 +333,146 @@ jobs:
-            - "~/ParlAI/data"
-      - store_test_results:
-          path: test-results
+      - runtests:
+          more_installs:
+            - installtorchgpu16
+          install_cuda: true
+          cachename: gpu16
+          marker: unit
 
   long_gpu_tests:
-    <<: *gpu
+    executor: gpu
     working_directory: ~/ParlAI
     parallelism: 8
     steps:
-      - checkout
-      - <<: *fixgit
-      - restore_cache:
-          key: nvidia-downloads-20200516
-      - <<: *setupcuda
-      - save_cache:
-          key: nvidia-downloads-20200516
-          paths:
-            - "~/nvidia-downloads/"
-      - restore_cache:
-          key: deps-20200921-nightly-{{ checksum "requirements.txt" }}
-      - <<: *setup
-      - <<: *installtorchgpu16
-      - <<: *installdeps
-      - save_cache:
-          key: deps-20200921-nightly-{{ checksum "requirements.txt" }}
-          paths:
-            - "~/venv/bin"
-            - "~/venv/lib"
-      - <<: *findtests
-      - restore_cache:
-          key: data-20200904-nightly-{{ checksum "teststorun.txt" }}
-      - run:
-          name: Nightly GPU tests
-          no_output_timeout: 60m
-          command: coverage run -m pytest --junitxml=test-results/junit.xml -m nightly_gpu -v -s
-      - <<: *codecov
-      - save_cache:
-          key: data-20200904-nightly-{{ checksum "teststorun.txt" }}
-          paths:
-            - "~/ParlAI/data"
-      - store_test_results:
-          path: test-results
-
-  quickstart_tests:
-    <<: *gpu
-    working_directory: ~/ParlAI
-    parallelism: 1
-    steps:
-      - checkout
-      - <<: *fixgit
-      - restore_cache:
-          key: nvidia-downloads-20200516
-      - <<: *setupcuda
-      - save_cache:
-          key: nvidia-downloads-20200516
-          paths:
-            - "~/nvidia-downloads/"
-      - restore_cache:
-          key: deps-20200921-nightly-{{ checksum "requirements.txt" }}
-      - <<: *setup
-      - <<: *installtorchgpu16
-      - <<: *installdeps
-      - save_cache:
-          key: deps-20200921-nightly-{{ checksum "requirements.txt" }}
-          paths:
-            - "~/venv/bin"
-            - "~/venv/lib"
-      - run:
-          name: Fake teststorun.txt
-          command: echo "quickstart" > teststorun.txt
-      - restore_cache:
-          key: data-20200904-quick-{{ checksum "teststorun.txt" }}
-      - run:
-          name: Quickstart unit tests (GPU; pytorch 1.4)
-          command: sh tests/test_quickstart.sh
-      - <<: *codecov
-      - save_cache:
-          key: data-20200904-quick-{{ checksum "teststorun.txt" }}
-          paths:
-            - "~/ParlAI/data"
-      - store_test_results:
-          path: test-results
-
-
+      - runtests:
+          more_installs:
+            - installtorchgpu16
+          install_cuda: true
+          cachename: nightly
+          marker: nightly_gpu
+          pytest_flags: -v -s
 
   crowdsourcing_tests:
-    <<: *standard_cpu37
+    executor: small_cpu37
```

Comment:
hmm - the change is just to conserve resources? If so, that's fine with me - you have a better sense of how much each size CPU can handle

Reply:
Yeah, just lowering our bill